pandas/io/packers.py

```diff
@@ -470,7 +469,7 @@ def encode(obj):
                     }
 
     elif isinstance(obj, (datetime, date, np.datetime64, timedelta,
-                          np.timedelta64, NaTType)):
```

Comment:
FYI: 

```
In [40]: np.timedelta64('nat') is pd.NaT
Out[40]: False

In [41]: pd.Timedelta(np.timedelta64('nat')) is pd.NaT
Out[41]: True
```

this is reason for the complex type checking around NaTs; numpy has a 'nat' like value embedded in np.timedelta64/np.datetime64 (however its not very useful).